lib/spring/configuration.rb

```diff
@@ -5,6 +5,12 @@ class << self
     attr_accessor :application_root
     attr_writer :quiet
 
+    attr_accessor :connect_timeout
+    @connect_timeout = 5
+
+    attr_accessor :boot_timeout
+    @boot_timeout = 20
```

Comment:
These instance variables aren't being correctly set currently (the accessors return nil)

```
`+': can't convert NilClass into an exact number (TypeError)
```
It works if you place them outside of the class methods

```
module Spring
  @connect_timeout = 5
  @boot_timeout = 20

  class << self
```

Reply:
Yeah, sorry, just did a quick patch without testing before leaving my desk. I think it should be good now.